torch/_refs/__init__.py

```diff
@@ -1403,6 +1404,25 @@ def _trunc_divide(
 #
 
 
+@register_decomposition(torch.ops.aten.addcdiv)
+@out_wrapper()
+@elementwise_type_promotion_wrapper(
+    type_promoting_args=("self", "tensor1", "tensor2"),
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT,
```

Comment:
Is this correct? 

Today `torch.addcdiv` will actually throw an error when given integer tensors because that behavior is deprecated as part of our changes to division in PyTorch, where it went from being C-style division to Python 3.0-like true division. We can actually update the behavior of `torch.addcdiv` in ATen at this point to perform true division and be consistent with the implementation here.

However, if `addcdiv` performs true division then I think it should implement `INT_TO_FLOAT` type promotion to be consistent with `true_divide`?